frontend/src/components/DebugInformation/DebugInformation.tsx

```diff
@@ -64,6 +64,7 @@ const propsToShow = [
   'authStrategy',
   'clusters',
   'gatewayAPIEnabled',
+  'gatewayAPIClasses',
```

Comment:
trivial - alphabetized out of order in a few places (not a blocker, can fix in some other PR)